addons/pos_self_order/static/src/Components/FloatingButton/FloatingButton.js

```diff
@@ -0,0 +1,13 @@
+/** @odoo-module */
+
+import { useSelfOrder } from "@pos_self_order/SelfOrderService";
+
+const { Component } = owl;
+
+export class FloatingButton extends Component {
+    static template = "FloatingButton";
+    setup() {
+        this.selfOrder = useSelfOrder();
+    }
+}
+export default { FloatingButton };
```

Comment:
Refrain from using `export default`. Just the basic export. This way, you'll always have to import like this:

```js
import { FloatingButton } from "./Components/FloatingButton/FloatingButton";
```